Contact.html

```diff
@@ -23,22 +23,22 @@
 <li><a href="MNBites.html">Market News Bites!</a></li>
 </ul>
     </div>
-    <div class="primary-content t-border">
-    <form>
+    <div class="primary-content2 t-border" id="contactpage">
+    <form class="contactus">
   First name:<br>
-  <input type="text" name="firstname"><br>
+  <input type="text" name="firstname"><br><br>
   Last name:<br>
-  <input type="text" name="lastname"><br>
+  <input type="text" name="lastname"><br><br>
   Email:<br>
-  <input type="text" name="email"><br>
+  <input type="text" name="email"><br><br>
   Phone Number:<br>
-  <input type="number" name="phone"><br>
+  <input type="tel" name="phone"><br><br>
   Message:<br>
-  <input type="text" name="message"><br>
-  <input type="submit" value="Submit">
-      
+   <textarea rows="6" cols="35" name="reply"></textarea> <br><br>
+       <input type="submit" value="Submit">
+ 
       </form>
-    
+    <img src="img/bull_vs_bear.jpg" alt="bullbear">

```

Comment:
move your img so that it's above
`<form class="contactus">` 
but below
`<div class="primary-content2 t-border" id="contactpage">`

related question: http://stackoverflow.com/questions/8287265/html-float-right-element-order